docs/ref/contrib/admin/actions.txt

```diff
@@ -149,6 +149,26 @@ That code will give us an admin change list that looks something like this:
 
 .. image:: _images/adding-actions-to-the-modeladmin.png
 
+.. versionchanged:: 6.0
+
+    The actions will be available also in the detail view but the description should be
+    adjusted to make sense we are talking about only one object::
```

Comment:
Reads better like this to my native English brain:

"The actions will also be available in the detail view but the description should be adjusted to show we are only talking about a single object."